src/Components/CheckBox/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { HTMLAttributes, VFC } from "react";
+
+import { Container, CheckMark, Input, Svg, Path } from "./style";
+
+interface CheckBoxProps extends HTMLAttributes<HTMLInputElement> {
+  checked: boolean;
+}
+
+const CheckBox: VFC<CheckBoxProps> = ({ ...props }) => (
```

Comment:
```suggestion
const CheckBox = props => (
```

spread operator가 필요없을 것 같아요~

Reply:
그러네요... 왜 이렇게 했을까요..ㅜ 
수정했습니다 ㅎㅎ
commit link : https://github.com/woowacourse/react-shopping-cart/pull/12/commits/ac4a0273a6fcbeda4218feb9e1fad079d8a77dba